src/Microsoft.IdentityModel.JsonWebTokens/JsonWebTokenHandler.cs

```diff
@@ -1096,8 +1096,8 @@ public virtual TokenValidationResult ValidateToken(string token, TokenValidation
 
         private TokenValidationResult ValidateTokenPayload(JsonWebToken jsonWebToken, TokenValidationParameters validationParameters)
         {
-            var expires = (jsonWebToken.ValidTo == null) ? null : new DateTime?(jsonWebToken.ValidTo);
-            var notBefore = (jsonWebToken.ValidFrom == null) ? null : new DateTime?(jsonWebToken.ValidFrom);
```

Comment:
Any idea what the reason was behind doing the datetime conversion before?

Reply:
Similar to the Guid comment, DateTime is a value type.

Reply:
As far as the conversion from `DateTime` to `DateTime?`, looks to only be to explicitly match the signature of some of the validators, which is not needed.